session/src/test/java/org/apache/iotdb/session/IoTDBSessionIT.java

```diff
@@ -351,6 +352,47 @@ public void test() throws ClassNotFoundException, SQLException,
     session.close();
   }
 
+  @Test
+  public void TestSessionInterfacesWithDisabledWAL()
+      throws StatementExecutionException, IoTDBConnectionException,
+          BatchExecutionException {
+    session = new Session("127.0.0.1", 6667, "root", "root");
+    try {
+      session.open();
+    } catch (IoTDBConnectionException e) {
+      e.printStackTrace();
+    }
+
+    session.setStorageGroup("root.sg1");
+    String deviceId = "root.sg1.d1";
+
+    IoTDBDescriptor.getInstance().getConfig().setEnableWal(false);
+    createTimeseries();
+
+    // test insert record
+    List<String> measurements = new ArrayList<>();
+    measurements.add("s1");
+    measurements.add("s2");
+    measurements.add("s3");
+    for (long time = 0; time < 100; time++) {
+      List<String> values = new ArrayList<>();
+      values.add("1");
+      values.add("2");
+      values.add("3");
+      session.testInsertRecord(deviceId, time, measurements, values);
```

Comment:
Do not use testInsertRecord, use insertRecord

Reply:
Fixed